tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -58,23 +64,37 @@ resources even further down the chain.
 
 ![Ownership card. CRDs from Supply Chain have yellow arrows. Knative Resources have orange arrows. Built-in resources have red arrows.](images/runtime-resources-crd-hierarchy.png)
 
-## <a id="knative-service-details"></a> Knative service details page
+## <a id="resource-details"></a> Resources details page
 
-To view details about your Knative services, select any resource that has a Knative Service type.
-In this page, additional information is available for Knative resources, including:
+To view details about your resources, select any resource of your preference and
```

Comment:
I'm wondering if this section has too much detail. The more detail, the harder it is to keep it from going stale. What are your thoughts about something like this?:

```
Title: Workload details page

To get more information about a particular workload, simply select it from the index table on the Runtime Resources tab's main page. You will then be taken to a page that provides details on the resource, including status, ownership, and resource-specific information.
```

Reply:
Hey Andrew! I picked that approach because the `Detail Page` looks similar to most of the resources we handle, I thought that introducing the two tables should be enough to give the user an idea about what to expect in each specific detail page and let us cover each resource on an easy way.

Reply:
I understand that we want to be specific here, but I'm wondering how much value it adds versus us having to go update the docs every time a table gets added/removed. It may not be something that is top of mind.

What are other people's thoughts here?

Reply:
Addressed, can you please re-review it? @menehune23 @chinigo

Reply:
Addressed, please consider resolved.